app/controllers/links_controller.rb

```diff
@@ -32,7 +32,7 @@ def external_url(type:, options: {})
   def documentation_url(section = "", options = {})
     root_url = options[:root_url] || "https://theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#"
     if section.empty?
-      "https://theforeman.org/documentation.html##{SETTINGS[:version].short}"
+      options[:root_url] || "https://theforeman.org/documentation.html##{SETTINGS[:version].short}"
     else
       root_url + section
     end
```

Comment:
`https://theforeman.org/documentation.html` is an old url that doesn't exist anymore, it redirects to the quickstart guide. 
I believe this whole condition should just be: `root_url + section` without checking if the section is empty (since that would just return the root_url).

Reply:
The idea was to create a default url, so the user will not be redirected to 404 page, if some pages did move. Especially for stale plugins.

Reply:
but right now if the section is empty it points to a stale url, while the base works just fine. And this function doesn't work for plugins anyways?

Reply:
Nice catch @tbrisker !
I would be ok with removing the section,
or replacing it with another url, e.g: https://theforeman.org/manuals/`version`

Reply:
After talking to @tbrisker, it looks like leftovers of our old documentation site structure. Now we can redirect to `https://theforeman.org/manuals/#{SETTINGS[:version].short}/index.html`. Meaning we can simplify the method to always return `root_url + section` (in case the section is empty - it will fall back to `root_url`, which is fine.